test/specs/modules/Popup/Popup-test.js

```diff
@@ -289,6 +289,20 @@ describe('Popup', () => {
     })
   })
 
+  describe('positionFixed', () => {
+    it(`is "false" by default`, () => {
+      wrapperMount(<Popup open />)
+
+      wrapper.should.have.prop('positionFixed', false)
```

Comment:
```suggestion
      wrapper.should.not.have.prop('positionFixed')
```

https://github.com/Semantic-Org/Semantic-UI-React/pull/3760/files#r317650640